services/user.py

```diff
@@ -0,0 +1,47 @@
+from db.models import User
+
+
+def create_user(
+        username: str,
+        password: str,
+        email: str = None,
+        first_name: str = None,
+        last_name: str = None
+) -> None:
+    user = User.objects.create_user(
```

Comment:
Use `get_user_model()` instead of `User` as mentioned in the checklist. Apply this change to all places